CountryPicker/CountryPickerTests/CountryPickerTests.swift

```diff
@@ -17,7 +17,7 @@ class CountryPickerTests: XCTestCase {
     override func setUp() {
         super.setUp()
         // 1. Given 2. Then
-        country = Country(countryCode: "IN")
+        country = Country(countryCode: "IN", dialingCode: "91")
```

Comment:
Isn't this dialingCode is redundant here ?

Reply:
Some test cases below check for the dialing code.